github/actions_secrets.go

```diff
@@ -179,90 +252,102 @@ type EncryptedSecret struct {
 //
-// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#delete-a-repository-secret
-func (s *ActionsService) DeleteRepoSecret(ctx context.Context, owner, repo, name string) (*Response, error) {
-	u := fmt.Sprintf("repos/%v/%v/actions/secrets/%v", owner, repo, name)
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#create-or-update-a-repository-secret
+func (s *ActionsService) CreateOrUpdateRepoSecret(ctx context.Context, owner, repo string, eSecret *EncryptedSecret) (*Response, error) {
+	url := fmt.Sprintf("repos/%v/%v/actions/secrets/%v", owner, repo, eSecret.Name)
+	return s.putSecret(ctx, url, eSecret)
+}
 
-	req, err := s.client.NewRequest("DELETE", u, nil)
-	if err != nil {
-		return nil, err
-	}
+// CreateOrUpdateRepoDependabotSecret creates or updates a repository secret with an encrypted value.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#create-or-update-a-repository-secret
+func (s *ActionsService) CreateOrUpdateRepoDependabotSecret(ctx context.Context, owner, repo string, eSecret *EncryptedSecret) (*Response, error) {
+	url := fmt.Sprintf("repos/%v/%v/dependabot/secrets/%v", owner, repo, eSecret.Name)
+	return s.putSecret(ctx, url, eSecret)
+}
 
-	return s.client.Do(ctx, req, nil)
+// CreateOrUpdateOrgSecret creates or updates an organization secret with an encrypted value.
+//
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#create-or-update-an-organization-secret
+func (s *ActionsService) CreateOrUpdateOrgSecret(ctx context.Context, org string, eSecret *EncryptedSecret) (*Response, error) {
+	url := fmt.Sprintf("orgs/%v/actions/secrets/%v", org, eSecret.Name)
+	return s.putSecret(ctx, url, eSecret)
 }
 
-// ListOrgSecrets lists all secrets available in an organization
-// without revealing their encrypted values.
+// CreateOrUpdateOrgDependabotSecret creates or updates an organization secret with an encrypted value.
 //
-// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#list-organization-secrets
-func (s *ActionsService) ListOrgSecrets(ctx context.Context, org string, opts *ListOptions) (*Secrets, *Response, error) {
-	u := fmt.Sprintf("orgs/%v/actions/secrets", org)
-	u, err := addOptions(u, opts)
-	if err != nil {
-		return nil, nil, err
-	}
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions/#create-or-update-an-organization-secret
+func (s *ActionsService) CreateOrUpdateOrgDependabotSecret(ctx context.Context, org string, eSecret *EncryptedSecret) (*Response, error) {
+	url := fmt.Sprintf("orgs/%v/dependabot/secrets/%v", org, eSecret.Name)
+	return s.putSecret(ctx, url, eSecret)
+}
 
-	req, err := s.client.NewRequest("GET", u, nil)
-	if err != nil {
-		return nil, nil, err
-	}
+// CreateOrUpdateEnvSecret creates or updates a repository secret with an encrypted value.
```

Comment:
Should this say "environment secret" instead of "repository secret"?
It looks like this may have been an pre-existing copy/paste mistake.